soccer/src/soccer/strategy/agent/position/waller.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <cmath>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+#include <spdlog/spdlog.h>
+
+#include <rj_msgs/action/robot_move.hpp>
+#include <rj_msgs/msg/empty_motion_command.hpp>
+
+#include "planning/instant.hpp"
+#include "position.hpp"
+#include "rj_common/field_dimensions.hpp"
+#include "rj_common/time.hpp"
+#include "rj_constants/constants.hpp"
+#include "rj_geometry/geometry_conversions.hpp"
+#include "rj_geometry/point.hpp"
+
+namespace strategy {
+
+/*
+ * The Waller role provides the implementation for a defensive robot that implements
+ * this class to have a waller-like behavior where it aims to serve as a wall between
+ * the ball and the goal.
+ */
+class Waller {
+public:
+    Waller(int wallerNum);
```

Comment:
```suggestion
    Waller(int waller_num);
```
local variables are snake_case